### PR TITLE
[Switch] Documentation and tester update for switch on Android

### PR DESCRIPTION
**Pull request by rohanprasadofficial**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [ ] win32 (Office)
- [ ] windows
- [x] android

### Description of changes

Updates the documentation and tester app for Link for android.

### Verification

Visual Verification on Android

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [x] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/experimental/Switch/src/Switch.types.ts [hunk 1]
```diff
@@ -110,6 +110,7 @@ export interface SwitchTokens extends LayoutTokens, FontTokens, IBorderTokens, I
 
   /**
    * States that can be applied to a switch
+   * Note: 'hovered','focused','before','beforeContent','above' are not supported for Android
```
Comment (rurikoaraki) [inline packages/experimental/Switch/src/Switch.types.ts:113]: Please update similar comments in SPEC.md

Comment (rohanprasadofficial) [reply packages/experimental/Switch/src/Switch.types.ts:113]: Updated.

File: packages/experimental/Switch/src/Switch.types.ts [hunk 2]
```diff
@@ -155,18 +156,21 @@ export interface SwitchProps extends Omit<PressablePropsExtended, 'onPress'> {
...
    * The Switch's text that shows when it is in a true state
+   * Note: Not supported for Android
    */
   onText?: string;
 
   /**
    * The position of the label relative to the Switch. The position value 'after' is mutually
    * exclusive with the onText and offText props. This is due to variable width
    * of the text props causing the Switch's position to change when it shouldn't.
+   * Note : 'before' , 'above' are not supported on Android
```
Comment (ankraj12) [inline packages/experimental/Switch/src/Switch.types.ts:173]: minor nit's on removing the extra spaces.

**Review by ankraj12 — APPROVED** [review]

**Review by rurikoaraki — APPROVED** [review]

File: packages/experimental/Switch/src/Switch.types.ts (hunk 2 above)
Comment (rohanprasadofficial) [reply packages/experimental/Switch/src/Switch.types.ts]: Removed.

**Pull request merged**